PR#2548 – Liquid Exception

- [ ] Liquid Exception: Liquid syntax error (line 54): Unknown tag 'January6DocumentDocumentLinks' in /home/runner/work/blog/blog/_posts/2025/12/2025-12-04-JimJordanScurriesIntoDarknessJackSmithTestimony.md

```
Run bundle exec jekyll build --trace --incremental --baseurl "/blog"
  bundle exec jekyll build --trace --incremental --baseurl "/blog"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_PAGES: true
    JEKYLL_ENV: production
    JEKYLL_GITHUB_TOKEN: 
    LOG_LEVEL: debug
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
Configuration file: /home/runner/work/blog/blog/_config.yml
            Source: /home/runner/work/blog/blog
       Destination: /home/runner/work/blog/blog/_site
 Incremental build: enabled
      Generating... 
  AI Related Posts: Creating cache [.ai_related_posts_cache.sqlite3]
  AI Related Posts: Fetch disabled. Using cached related posts data.
  AI Related Posts: Found 0 cached embeddings; skipped 1412 fetches
       Jekyll Feed: Generating feed for posts
         AutoPages: Disabled/Not configured in site.config.
        Pagination: ----------------------------
        Pagination: Page: blogpager.html
        Pagination:  Active configuration
        Pagination:   Enabled:          true
        Pagination:   Items per page:   15
        Pagination:   Permalink:        /page/:num/
        Pagination:   Title:            :title - page :num of :max
        Pagination:   Limit:            0
        Pagination:   Sort by:          date
        Pagination:   Sort reverse:     true
        Pagination:  Active Filters
        Pagination:   Collection:       posts
        Pagination:   Offset:           0
        Pagination:   Category:         [Not set]
        Pagination:   Tag:              [Not set]
        Pagination:   Locale:           [Not set]
        Pagination:  Filtering by: Category  1412 => 1412
        Pagination:  Filtering by: Tag       1412 => 1412
        Pagination:  Filtering by: Locale    1412 => 1412
        Pagination: Rolling through the date fields for all documents
        Pagination: Complete, processed 1 pagination page(s)
  Liquid Exception: Liquid syntax error (line 54): Unknown tag 'January6DocumentDocumentLinks' in /home/runner/work/blog/blog/_posts/2025/12/2025-12-04-JimJordanScurriesIntoDarknessJackSmithTestimony.md
bundler: failed to load command: jekyll (/home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/bin/jekyll)
/home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/document.rb:23:in 'Liquid::Document#unknown_tag': Liquid syntax error (line 54): Unknown tag 'January6DocumentDocumentLinks' (Liquid::SyntaxError)
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/document.rb:11:in 'block in Liquid::Document#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:32:in 'Liquid::BlockBody#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/document.rb:10:in 'Liquid::Document#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/document.rb:5:in 'Liquid::Document.parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/template.rb:130:in 'Liquid::Template#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/template.rb:114:in 'Liquid::Template.parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:13:in 'block in Jekyll::LiquidRenderer::File#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:70:in 'Jekyll::LiquidRenderer::File#measure_time'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:12:in 'Jekyll::LiquidRenderer::File#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:124:in 'Jekyll::Renderer#render_liquid'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:80:in 'Jekyll::Renderer#render_document'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:63:in 'Jekyll::Renderer#run'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:572:in 'Jekyll::Site#render_regenerated'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:557:in 'block (2 levels) in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'block in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Hash#each_value'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:210:in 'Jekyll::Site#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:80:in 'Jekyll::Site#process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:28:in 'Jekyll::Command.process_site'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:65:in 'Jekyll::Commands::Build.build'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:36:in 'Jekyll::Commands::Build.process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'block in Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:18:in 'block (2 levels) in Jekyll::Commands::Build.init_with_program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'block in Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in 'Mercenary::Program#go'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in 'Mercenary.program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/exe/jekyll:15:in '<top (required)>'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/bin/jekyll:25:in 'Kernel#load'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/bin/jekyll:25:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli/exec.rb:59:in 'Kernel.load'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli/exec.rb:59:in 'Bundler::CLI::Exec#kernel_load'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli/exec.rb:23:in 'Bundler::CLI::Exec#run'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli.rb:452:in 'Bundler::CLI#exec'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor/command.rb:28:in 'Bundler::Thor::Command#run'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in 'Bundler::Thor::Invocation#invoke_command'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor.rb:538:in 'Bundler::Thor.dispatch'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli.rb:35:in 'Bundler::CLI.dispatch'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor/base.rb:584:in 'Bundler::Thor::Base::ClassMethods#start'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli.rb:29:in 'Bundler::CLI.start'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/gems/3.4.0/gems/bundler-2.6.9/exe/bundle:28:in 'block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/friendly_errors.rb:117:in 'Bundler.with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/gems/3.4.0/gems/bundler-2.6.9/exe/bundle:20:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/bin/bundle:25:in 'Kernel#load'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/bin/bundle:25:in '<main>'
Error: Process completed with exit code 1.
```

_Originally posted by @RalphHightower in https://github.com/RalphHightower/blog/issues/2548#issuecomment-3614593167_

--- a/_posts/2025/12/2025-12-04-JimJordanScurriesIntoDarknessJackSmithTestimony.md
+++ b/_posts/2025/12/2025-12-04-JimJordanScurriesIntoDarknessJackSmithTestimony.md
@@ -63,7 +63,7 @@ In October, [Smith](https://www.justice.gov/archives/sco-smith) notified the [Ho
 
 [Smith](https://www.justice.gov/archives/sco-smith) clearly wanted more transparency, but he wasn’t in a position to dictate the terms.
 
-[The New York Times reported](https://www.nytimes.com/live/2025/12/03/us/trump-news/c665b349-d12c-56d9-a879-456e05554314?smid=url-share) that [House](https://www.house.gov/) [Republicans](https://www.gop.com/) “have been reluctant to give [Smith](https://www.justice.gov/archives/sco-smith)] a prime public platform out of concern that he could embarrass [Trump](https://www.donaldjtrump.com/) by making a compelling case for the indictments over the [president](https://www.whitehouse.gov/)’s attempt to overturn the [2020 election]({% January6DocumentDocumentLinks.md %}) and his retention of classified documents.”
+[The New York Times reported](https://www.nytimes.com/live/2025/12/03/us/trump-news/c665b349-d12c-56d9-a879-456e05554314?smid=url-share) that [House](https://www.house.gov/) [Republicans](https://www.gop.com/) “have been reluctant to give [Smith](https://www.justice.gov/archives/sco-smith)] a prime public platform out of concern that he could embarrass [Trump](https://www.donaldjtrump.com/) by making a compelling case for the indictments over the [president](https://www.whitehouse.gov/)’s attempt to overturn the [2020 election]({% kink January6DocumentDocumentLinks.md %}) and his retention of classified documents.”
 
 Those concerns give away the game.
 

